pkg/jobs/resultcols.go

```diff
@@ -26,6 +26,16 @@ var BulkJobExecutionResultHeader = colinfo.ResultColumns{
 	{Name: "bytes", Typ: types.Int},
 }
 
+// OnlineRestoreJobExecutionResultHeader is the header for an online restore
+// job, which provides a header different from the usual bulk job execution
+var OnlineRestoreJobExecutionResultHeader = colinfo.ResultColumns{
+	{Name: "job_id", Typ: types.Int},
+	{Name: "tables", Typ: types.Int},
+	{Name: "approx_rows", Typ: types.Int},
+	{Name: "approx_bytes", Typ: types.Int},
+	{Name: "background_download_job_id", Typ: types.Int},
```

Comment:
Do the results in the sql shell stay in one line? If not, we should remove `background_` from the column name

Reply:
It does stay in one line for me:
<img width="1222" alt="image" src="https://github.com/cockroachdb/cockroach/assets/39608887/628164a3-15be-4e06-8582-9aa8e0c12658">

Reply:
nit: your sql shell format is comic sans :D

Reply:
lol everything is comic sans for me -- technically it's [Comic Code](https://tosche.net/fonts/comic-code), a monospaced comic sans XD